base/char.jl

```diff
@@ -76,9 +80,18 @@ end
 
 Return the Unicode codepoint (an unsigned integer) corresponding
 to the character `c` (or throw an exception if `c` does not represent
-a valid character). For `Char`, this is a `UInt32` value, but
+a well-formed character). For `Char`, this is a `UInt32` value, but
```

Comment:
Would this be equivalent?
```suggestion
to the character `c` (or throw an exception if `c` represents
a malformed character). For `Char`, this is a `UInt32` value, bu
```
If so, I think it is more clear because `malformed` more clearly refers to `ismalformed`.

Reply:
Done.